mirrord/layer/src/error.rs

```diff
@@ -338,3 +338,9 @@ impl From<HookError> for *mut c_char {
         ptr::null_mut()
     }
 }
+
+impl From<frida_gum::Error> for LayerError {
```

Comment:
Why manual impl? isn't this the same as `#[from]`

Reply:
`#[from]` requires `impl std::error::Error` and frida-rust was changed to be no-stdlib by default, so its errors don't provide that.